src/main/java/mate/jdbc/util/ConnectionUtil.java

```diff
@@ -0,0 +1,27 @@
+package mate.jdbc.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    static {
+        try {
+            Class.forName("com.mysql.cj.jdbc.Driver");
```

Comment:
Using Class.forName() is no longer required with newer JDBC 4.x compliant drivers and Java 6 or higher. The DriverManager can automatically load the necessary driver based on the JDBC URL.